tensorflow-framework/src/main/java/org/tensorflow/framework/metrics/impl/MetricVariable.java

```diff
@@ -78,8 +79,8 @@ public MetricVariable(
       } else {
         throw new IllegalArgumentException(
             String.format(
-                "An initializer for variable %s of type %s is required",
-                variable.toString(), type.getSimpleName()));
+                "Type %s is not a supported for metric variables",
```

Comment:
"Type %s is not **a** supported for metric variables" -> "Type %s is not supported for metric variables"

Reply:
OK